.spacelift/config.yml

```
@@ -1,5 +1,5 @@
 version: 1
-module_version: 2.1.0
+module_version: 2.1.1
```

This change is introducing a new functionality and I think updating the minor version would be more appropriate. Thank you!

```suggestion
module_version: 2.2.0
```